### PR TITLE
add h2 in index

### PR DESCRIPTION
**Pull request by jhp0621**
Issues addressed:

### Items Addressed
- https://app.asana.com/0/1203996148278240/1203996148278244/f

### Code Review Questions
- General review

## Author Checklist
- [ ] Assign a dev reviewer when you're ready for code review

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import "./styles.css";
 
 document.getElementById("app").innerHTML = `
 <h1>Hello Vanilla!</h1>
+<h2>This is to create a pull request to see if the event handler gets triggered!</h2>
```
Comment (katespace) [inline src/index.js:5]: leaving a comment

Comment (jhp90621) [reply src/index.js:5]: another

**Review by katespace — CHANGES_REQUESTED** [review]
request changes

**Review by jhp0621 — COMMENTED** [review]
edit comment

**Review by katespace — CHANGES_REQUESTED** [review]
requesting changes!

**Review by katespace — COMMENTED** [review]
leaving a comment

**Review by katespace — APPROVED** [review]
LGTM

**Review by katespace — APPROVED** [review]
great job!

**Pull request merged**